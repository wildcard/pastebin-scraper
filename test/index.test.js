const { scrapeIndexPage } = require("../scrape");

test("Run", async () => {
    await scrapeIndexPage(3);
});
