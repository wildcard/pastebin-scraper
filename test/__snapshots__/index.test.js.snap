// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parsing Lite Table 1`] = `
[
  {
    "id": "6ryvL9V2",
    "syntax": "TypeScript",
    "title": "AuthGuard",
  },
  {
    "id": "jWHZjpCC",
    "syntax": "Rust",
    "title": "grid paths",
  },
  {
    "id": "Lg2LV0a0",
    "syntax": "Bash",
    "title": "fdsafsd",
  },
  {
    "id": "pJEPaduj",
    "syntax": "Python",
    "title": "Untitled",
  },
]
`;

exports[`Parsing Lite Table with limit=2 1`] = `
[
  {
    "id": "6ryvL9V2",
    "syntax": "TypeScript",
    "title": "AuthGuard",
  },
  {
    "id": "jWHZjpCC",
    "syntax": "Rust",
    "title": "grid paths",
  },
]
`;
